crates/ruma/Cargo.toml

```diff
@@ -191,6 +191,7 @@ unstable-msc4171 = ["ruma-events?/unstable-msc4171"]
 unstable-msc4186 = ["ruma-client-api?/unstable-msc4186"]
 unstable-msc4203 = ["ruma-appservice-api?/unstable-msc4203"]
 unstable-msc4230 = ["ruma-events?/unstable-msc4230"]
+unstable-msc4274 = ["ruma-events?/unstable-msc4274"]
```

Comment:
It needs to be added to `__unstable-mscs` too.

Reply:
Thanks, I wasn't aware of this.